wpilibj/src/athena/java/edu/wpi/first/wpilibj/I2C.java

```diff
@@ -93,54 +94,63 @@ public synchronized boolean transaction(byte[] dataToSend, int sendSize, byte[]
   /**
    * Generic transaction.
    *
-   * This is a lower-level interface to the I2C hardware giving you more control
-   * over each transaction.
+   * <p>This is a lower-level interface to the I2C hardware giving you more control over each
+   * transaction.
    *
-   * @param dataToSend Buffer of data to send as part of the transaction. Must
-   *        be allocated using ByteBuffer.allocateDirect().
-   * @param sendSize Number of bytes to send as part of the transaction.
-   * @param dataReceived Buffer to read data into. Must be allocated using
-   *        ByteBuffer.allocateDirect().
-   * @param receiveSize Number of bytes to read from the device.
+   * @param dataToSend   Buffer of data to send as part of the transaction. Must be allocated using
+   *                     ByteBuffer.allocateDirect().
+   * @param sendSize     Number of bytes to send as part of the transaction.
+   * @param dataReceived Buffer to read data into. Must be allocated using ByteBuffer
+   *                     .allocateDirect().
```

Comment:
Line break in middle of ClassName.funcName()?  Won't this add incorrect whitespace in the output?